strongbox-user-management/src/main/java/org/carlspring/strongbox/users/domain/Features.java

```diff
@@ -0,0 +1,62 @@
+package org.carlspring.strongbox.users.domain;
+
+import org.carlspring.strongbox.data.domain.GenericEntity;
+
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+
+import com.google.common.base.Objects;
+
+/**
+ * Additional user settings (for example, per-repository access settings).
+ *
+ * @author Alex Oreshkevich
+ */
+public class Features
```

Comment:
This is not a good name for the class. "Features" is very broad/generic and, at first glance, other developers won't be able to understand what this refers to.